ulauncher/api/server/ExtensionDownloader.py

```diff
@@ -182,7 +182,20 @@ def unzip(filename: str, ext_path: str) -> None:
     temp_ext_path = mkdtemp(prefix='ulauncher_dl_')
 
     with ZipFile(filename) as zipfile:
-        zipfile.extractall(temp_ext_path)
+        for zipinfo in zipfile.infolist():
+            filepath = zipfile.extract(zipinfo, temp_ext_path)
+            # Check if file is symlink from its attributes or zipfile did extract it as a symlink
+            # see https://learning-python.com/cgi/showcode.py?name=ziptools/ziptools/ziptools/zipsymlinks.py
+            if (zipinfo.external_attr >> 28) != 0xA or os.path.islink(filepath):
+                continue
+            with open(filepath) as f:
```

Comment:
I might be concerned about whether or not this is a directory, but I think the symlink check above is sufficient to be sure that if we get here what we have is a small text file containing the path to point to. :+1:

Reply:
Directories return False for (zipinfo.external_attr >> 28) != 0xA.

Actually my first tries had a safeguard like this
```python
# zipinfo.is_dir(), only python >= 3.6
if (zipinfo.external_attr >> 28) != 0xA or zipinfo.is_dir() or os.path.islink(filepath):  
    ...
```